inductiva/resources/machine_groups.py

```diff
@@ -69,13 +69,25 @@ def _machine_group_list_to_str(machine_group_list) -> str:
             spot, machine_group.create_time
         ])
 
-    formatters = {"Started at (UTC)": format_utils.datetime_formatter}
-
-    return format_utils.get_tabular_str(
-        rows,
-        columns,
-        formatters=formatters,
-    )
+    formatters = {
+        "Started at (UTC)": [format_utils.datetime_formatter],
+        "Name": [format_utils.spacing_formater],
+        "Machine Type": [format_utils.spacing_formater],
+        "Elastic": [format_utils.spacing_formater],
+        "Type": [format_utils.spacing_formater],
+        "# machines": [format_utils.spacing_formater],
+        "Disk Size in GB": [format_utils.spacing_formater],
+        "Spot": [format_utils.spacing_formater],
+    }
+    header_formatters = [lambda x: x.upper()]
```

Comment:
This seems to be a general formatter we will apply to all tables, hence it should be set by default in the `get_tabular_str`.

Reply:
I am not sure we should add default formatters to the `get_tabular_str`